spec/integration/mixed_collection_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+RSpec.describe JSONAPI::Serializer do
+  let(:car) { Car.fake }
+  let(:bus) { Bus.fake }
+  let(:truck) { Truck.fake }
```

Comment:
It would be nice to reuse existing serializers. Consider using the `Movie`, `Actor`, `User` for tests.

Reply:
Sure thing.